scripts/gen.py

```diff
@@ -24,11 +25,11 @@ class ArgTemplate(string.Template):
   idpattern = r'[a-z0-9_]+'
 
 
-FuncDef = namedtuple_with_defaults('FuncDef', 'cpp_sig, aten_sig')
+FuncDef = namedtuple_with_defaults('FuncDef', 'cpp_sig, aten_sig, compound')
```

Comment:
I am debating whether we should expand the JSON data or just stick it here as a whole (that excludes aten_sig).

Reply:
I feel it's good to have them in separate fields given that they're indeed func properties?

Reply:
SG